.github/workflows/dev-docker-publish.yml

```diff
@@ -0,0 +1,48 @@
+name: Dev build and publish Docker distributions to Github Container Registry ghcr.io
+
+on:
+  push:
+    branches-ignore:
+      - master
+      - version-*
+    paths:
+      - gradle.properties
+#    - package_info.json
```

Comment:
Remove these lines

Reply:
It will be corrected after branches are synchronized